frontend/pages/projects/new.tsx

```diff
@@ -17,6 +17,11 @@ import { GroupedEnums } from 'types/enums';
 import { useCreateProject } from 'services/account';
 import { getEnums } from 'services/enums/enumService';
 
+const PROJECT_QUERY_PARAMS = {
+  // We set the `locale` as `null` so that we get the project in the account's language instead of the UI language
+  locale: null,
+};
+
```

Comment:
```suggestion

```

Reply:
If we create the project without locale it creates a bug. The locale should be the project developer’s language. 

UPDATE: I asked Martin Tomas and he confirmed that we should send the locale with the project developer’s language